dask_kubernetes/core.py

```diff
@@ -156,7 +163,7 @@ def __init__(
         env=None,
         auth=ClusterAuth.DEFAULT,
         asynchronous=False,
-        **kwargs
+        **kwargs,
```

Comment:
This will kill Python 3.5 compatibility.  This is likely a black issue.  You might want to update to black master and make sure that the rest of your file is compatible with Python 3.5.  If that's the case then black won't put these in there (I ran into this today as well, and found it frustrating).

Reply:
Thanks for the note!